.github/workflows/deploy.yml

```
@@ -33,7 +33,7 @@ jobs:
         with:
           repository: AdobeDocs/adp-devsite-scripts
           path: scripts
-          ref: main
+          ref: deploy-fail
```

is this left over from testing?


```suggestion
          ref: main
```

---

Ah yes, it should point to main now.